engines/pytorch/pytorch-engine/README.md

```diff
@@ -94,7 +94,7 @@ installed on your GPU machine, you can use one of the following library:
 
 #### Linux GPU
 
-- ai.djl.pytorch:pytorch-jni:1.10.0-0.16.0
+- ai.djl.pytorch:pytorch-jni:1.10.0-0.17.0
```

Comment:
```suggestion
- ai.djl.pytorch:pytorch-jni:1.11.0-0.17.0
```

Reply:
We need release 1.11.0 binary first.

Reply:
I think I will finish the binary release (or testing it) with the 0.17.0 release